components/Chips/src/MDCChipView.m

```diff
@@ -565,6 +573,25 @@ - (void)updateTitleColor {
   self.titleLabel.textColor = [self titleColorForState:self.state];
 }
 
+- (void)updateAccessibility {
+
+  // Clearing and then adding the relevant traits based on current the state (while accommodating concurrent states).
+  self.accessibilityTraits &= !UIAccessibilityTraitSelected;
```

Comment:
Please use bitwise negation instead of boolean negation. I believe this could be

```objectivec
self.accessibilityTraits &= ~(UIAccessibilityTraitSelected | UIAccessibilityTraitNotEnabled);
```